src/secp256k1/libsecp256k1.pc.in

```diff
@@ -5,7 +5,7 @@ includedir=@includedir@
 
 Name: libsecp256k1
 Description: Optimized C library for EC operations on curve secp256k1
-URL: https://github.com/bitcoin/secp256k1
+URL: https://github.com/bitcoin-core/secp256k1
```

Comment:
Once again, this is where the dev happens. Why does this need a fix ? What does this needs to be fixed to ?